sources/packages/backend/apps/api/src/services/student-scholastic-standings/_tests_/unit/student-scholastic-standings.service.adjustStudyBreaks.spec.ts

```diff
@@ -0,0 +1,108 @@
+import { StudyBreak } from "@sims/sims-db";
+import {
+  StudentRestrictionService,
+  StudentScholasticStandingsService,
+} from "../../..";
+import {
+  NotificationActionsService,
+  StudentRestrictionSharedService,
+} from "@sims/services";
+import { DataSource } from "typeorm";
+import { createMock } from "@golevelup/ts-jest";
+import { dateDifference } from "@sims/utilities";
+import { OFFERING_STUDY_BREAK_MAX_DAYS } from "../../../../utilities";
+
+class PublicClassToTestAdjustStudyBreak extends StudentScholasticStandingsService {
```

Comment:
I can see this still as an attempt to expose a private method to be "unit tested".
As a general rule, the intention is to try to avoid changing the production code in a way that would serve only for the unit tests.